### DISCUSSION TITLE
[Request] "All Notes" link in Week view

### DISCUSSION THREAD

**Discussion by vinabqrq** [kudrykv/latex-yearly-planner#62] [Ideas]
Hey. Thanks for your hard work, this is an amazing tool. Have been using on my Boox Note Air 2 for quite some time now.

Starting to plan for next year, I would like to request something I have been missing: an "All Notes" link on the week view and on the Month ciew. Is this something simple to add?

This is more a recommendation that I think could help all templates, but if it is too specific of a request and you don't want to change all of them, I would really appreciate a RM Breadcrumb Lined Default.

Thanks again for your work!

**Comment by kudrykv** [comment]
> something I have been missing: an "All Notes" link on the week view and on the Month ciew. Is this something simple to add?

Yep, that should be pretty straightforward. The "hard part" is to recompile all the templates :) That might take a few hours, plus another bit to sort them out.

I'll try to look into it. Maybe, I'll just dump everything underneath here

**Reply by vinabqrq** [reply]
Thanks! I appreciate it a lot. :)
